tests/filtered_relation/tests.py

```diff
@@ -40,7 +40,7 @@ def setUpTestData(cls):
 
     def test_filtered_relation_select_related(self):
         qs = Author.objects.annotate(
-            book_join=FilteredRelation('book', condition=Q()),
```

Comment:
There is a trade-of, which is making the API less explicit.
Remember all those `PathInfo` namedtuples ? we decided to replace positional arguments by named ones.
It looks to me this changeset goes against this philosophy.
Also I tend to believe reading matters most than writing.

Reply:
I think this is fine as it's two arguments, and the class name gives the reader a clue too - this `Q()` is the filter for the relation :) Additionally we have to test with pos args because users will use it like that. There should really be a test with a keyword arg too though.

Reply:
An alternative is to make the signature `__init__(self, relation_name, *, condition=Q())` to force passing the `Q` as a kwarg. This could be advantageous if more kwargs are added later.